src/mod_last.erl

```diff
@@ -46,6 +46,7 @@
          session_cleanup/5,
          remove_domain/3,
          remove_unused_backend_opts/1]).
+-export([user_receive_iq/3]).
```

Comment:
Minor: why is this one exported separately? The above are already IQ and hook handlers (two types of handlers), and they are exported together. So it looks like a minor inconsistency.